src/utils/jump_utils.jl

```diff
@@ -70,16 +70,22 @@ end
 
 function get_column_names(
     ::OptimizationContainerKey,
-    arr::DenseAxisArray{T, 2, K},
+    array::DenseAxisArray{T, 2, K},
 ) where {T, K <: NTuple{2, Any}}
-    return axes(arr)[1]
+    return axes(array)[1]
+end
+
+function _get_column_names(
+    arr::SparseAxisArray{T, N, K},
+)::Vector{String} where {T, N, K <: NTuple{N, Any}}
```

Comment:
Do you need `::Vector{String}`? Isn't `encode_tuple_to_column` guaranteed to produce strings?

Reply:
I added it because `@code_warntype` wasn't detecting the return type. I could remove it.

Reply:
fixed via 23c1f56